README.md

```diff
@@ -57,12 +57,38 @@ This project supports a basic [Vagrant](http://docs.vagrantup.com/v2/getting-sta
 
 1. Run vagrant up command
 
-    vagrant up
+        vagrant up
+
 
 2. Visit [http://localhost:8085](http://localhost:8085) in your browser
 
 Look in [Vagrantfile](Vagrantfile) for configuration details.
 
+### Docker server
+
+This project supports a basic [Docker](https://www.docker.com/whatisdocker/) configuration
+
+1. [Install Docker](http://docs.docker.com/installation/)
+2. `cd` into the directory of the project
+3. Build the docker image based on the *Dockerfile*:
+        
+        docker build .
```

Comment:
Fenced code blocks are not in the [official spec](http://daringfireball.net/projects/markdown/); they are an extension, although a fairly common one.

If a code block is within a list, then it needs to be indented by eight spaces. First four are for the list continuation and the second four are for the code block.